### PR TITLE
Login Event Handler

### PR DESCRIPTION
**Pull request by Dippys**
With some example code too.

### CODE REVIEW

**Comment by Dippys** [comment]
@billsonnn will this be reviewed? It's been sitting stale for 2 weeks now

File: Turbo/Extensions/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -66,6 +66,7 @@ public static void AddNetworking(this IServiceCollection services)
             // Event Handlers
             services.AddTransient<IEventHandlerManager, EventHandlerManager>();
             services.AddTransient<ITestEventHandler, TestEventHandler>();
+            services.AddTransient<ITestEventHandler, UserLoginEventHandler>();
```
Comment (dotmbf) [inline Turbo/Extensions/ServiceCollectionExtensions.cs]: should use an explicit interface for this event

**Comment by dotmbf** [comment]
hey added a comment on ur pr

File: Turbo/Extensions/ServiceCollectionExtensions.cs (hunk 1 above)
Comment (Dippys) [reply Turbo/Extensions/ServiceCollectionExtensions.cs]: Fixed in latest commit

**Review by billsonnn — APPROVED** [review]

**Pull request merged**